fs/exists_test.ts

```diff
@@ -6,128 +6,445 @@ import { exists, existsSync } from "./exists.ts";
 const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
 const testdataDir = path.resolve(moduleDir, "testdata");
 
+Deno.test("[fs] existsNotExist", async function () {
+  const tempDirPath: string = await Deno.makeTempDir();
+  try {
+    assertEquals(await exists(path.join(tempDirPath, "not_exists")), false);
+  } catch (error) {
+    throw error;
```

Comment:
This catch block seems unnecessary (if you don't catch it, it's thrown as is)

(Can you also remove the similar blocks in other test cases?)